cylc/flow/scheduler.py

```diff
@@ -1774,6 +1775,8 @@ async def main_loop(self) -> None:
                     with suppress(KeyError):
                         self.timers[self.EVENT_STALL_TIMEOUT].stop()
 
+                self.update_data_store()
```

Comment:
Don't think this is needed.. As there's nothing in this block that requires data-store change, and if something else changes downstream it should message the data-store with a delta.